decidim/decidim
# Remove show statistics boolean fields in assemblies and processes

#### :tophat: What? Why?
Since Decidim 0.28, we have 2 n ew content blocks that we can configure for `assemblies` and `participatory process`. If you enable those content blocks, you may get in a situation when you enable the content block, yet the stats to not be shown, as the admin needs to go in the edit assembly admin page, where you would need to enable `Show statistics` 

#### Testing
1. Login as admin on localhost, and navigate to admin panel 
2. On an existing assembly, edit the assembly and uncheck `Show statistics`, then save
3. In the Landing page, admin make sure you have statistics content block enabled 
4. See the stats are not being displayed. 
5. in the content blocks disable stats, and in the admin enable `show statistics` 
6. See there are no stats displayed. 
7. Apply patch, and see the show statistics button is gone
8. See the display control on this matter is being handled only by content block. 

Additionally, on Processes you would have the show metrics checkbox that has been removed. 
1. Visit the participatory process admin page 
2. Uncheck show metrics & save the form 
3. Visit the Landing page area & enable the metrics content block 
4. Visit frontend Main page & see the metrics are being displayed. 

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!
